apache/superset
# chore(translations): German translation update

### SUMMARY
messages.po file for German (de) translation is updated.

### TESTING INSTRUCTIONS
Change language settings to German and check translations via GUI or manually check them in messages.po file.

### ADDITIONAL INFORMATION
